src/server/routes/entity/creator.js

```diff
@@ -232,18 +229,14 @@ router.get(
 			creator
 		);
 
-		const props = generateProps(req, res, {
-			creator,
```

Comment:
Looks like `creator` may be missing from the new set of props.

Reply:
Yeah, the above solves this problem as well.